### ISSUE TITLE
Request: show all photo directions on map

### ISSUE THREAD

**Issue by pbb72** [mapillary/mapillary_issues#2950]
The map currently only shows photo positions, but using a small circle, on the map. To see in what direction photos have been taken, each photo needs to be clicked.

If instead of circles, some triangular icon was used to show the photo direction, it would be much easier to quickly review in what directions new photos need to be submitted.

**Comment by Kaldea** [comment]
Hey Peter--- 

I'm sending these 8 tickets forward to the design and dev team to look into. I'm going to close these but will have the team working on them in internally. Thank you so much for taking the time to get this feedback to us on all of these topics!

**Comment by pbb72** [comment]
Hi @Kaldea.

This is not how bug reporting works.

You close issues when they are either fixed, or when it has been decided they will not be fixed. It's not expected to close issues simply because they are forwarded.

This can be interpreted as wanting to silence a contributor, especially when you're doing this to all issues one person posts.

Also for your organisation this can be counterproductive, since other users may discover the same problems, and report them in new issues because these closed reports don't show up in search results.

I can assume you will not follow up on my reports, as you have already closed them. All the others without even a single comment. This does not stimulate me to file more reports, while there are still more than enough problems.

Maybe, that is the whole reason for just shutting my reports.

**Comment by Kaldea** [comment]
Hi @pbb72 

Sorry about the confusion here. We're actually in the middle of trying to restructure how we receive these issues and requests and I may have jumped the gun here before we have things finalized. I will be reopening these here for the time being. Shortly we will have more information on the future of how reports will be sent in, as well as opening them up to the community for discussion.

Thanks

**Comment by pbb72** [comment]
Thanks @Kaldea! Sorry for flaring up. It's happened before on Mapillary, that my bug reports went without reply for long times, and then just got closed for unclear reasons. Looking forward to hearing more! 😃

**Comment by Kaldea** [comment]
I definitely understand :) My main focus right now in joining the team is to get these things answered faster, have things opened up to more eyes of both users and us here at Mapillary so we can start making better progress. Hang tight and I will make sure to reach out to you directly when we've sorted out exactly what the new workflow will look like!

**Comment by angelday** [comment]
> If instead of circles, some triangular icon was used to show the photo direction, it would be much easier to quickly review in what directions new photos need to be submitted.

The sequence editor is the intended place for working with directions.

<img width="777" alt="screen shot 2018-07-26 at 14 53 40" src="https://user-images.githubusercontent.com/50189/43263508-bd98454c-90e3-11e8-9c23-9c9315a1a4cd.png">

**Comment by voschix** [comment]
> The sequence editor is the intended place for working with directions.
>
I also have requested this feature.
It is not only useful in the context of editing, but also in any situation
where you plan your picture taking in advance to complete Mapillary
coverage. In most cases it is useful to have photos in forward and backward
direction (unless you use 360° cameras) .
I often plan may local trips also with the aim of "completing the map"
Mapillary-wise. I try to select routes that I have not taken during the
last year and where photos are not yet available in both directions.

**Comment by pbb72** [comment]
Agreed, this is not about _working_ with directions, this is about getting a good overview over the covered directions. In a dense urban area, sideways (90 degrees) photography is often more valuable than simply forwards, showing store fronts, side streets, etc. Being able to see all directions in an area, helps to plan new photo sessions. (Note also that the sequence editor ONLY shows the directions of one, the current, sequence!)

**Comment by angelday** [comment]
Ok, thanks guys for the insight.

I'm flagging this as a discussion that we'll keep open and move to the forum. (I'm sure you're aware, but if you're not, then here is the thing: we'll be sunsetting this Github repository.)
